model/registry/rpc.yaml

```diff
@@ -264,23 +264,13 @@ groups:
         type: int
         stability: experimental
         brief: "Uncompressed size of the message in bytes."
-      - id: client.request.body.size
+      - id: request.body.size
         type: int
         stability: experimental
-        brief: "The size of the client request payload body in bytes."
+        brief: "The size of the request payload body in bytes. For gRPC, this value is calculated through protobuf (including unary and streaming calls)."
```

Comment:
you can add grpc-specific note on grpc only by referencing that attribute and updating the brief in 

https://github.com/open-telemetry/semantic-conventions/blob/d182f9e66f48d854e8afcc74a0962bad89f9dd3c/model/trace/rpc.yaml#L61-74

Also, please provide the description of what the size is (not how it's calculated). If there is some protobuf documentation that's useful to link, let's link it.

Reply:
I add some information but only found link related to Java, I cound't find a description of the `body.size` on the protobuf [official website](https://protobuf.dev/).